tests/js/client/server_parameters/test-export-shard-usage-metrics-bytes-per-shard-per-user-cluster.js

```diff
@@ -139,15 +139,6 @@ function BaseTestSuite(targetUser) {
   };
 
   return {
-    setUpAll : function () {
```

Comment:
Note to reviewer:
This registered setUpAll twice, causing issues, moved the logic here into the other setUpAll